modules/rosa-sts-byo-oidc.adoc

```diff
@@ -60,25 +60,26 @@ I: Created OIDC provider with ARN 'arn:aws:iam::4540112244:oidc-provider/dvbwgdz
 +
 When creating your cluster, you must supply the OIDC config ID. The CLI output provides this value for `--mode auto`, otherwise you must determine these values based on `aws` CLI output for `--mode manual`.
 
-* Optional: you can save the OIDC configuration ID as a variable to use later. Run the following command to save the variable:
+** Optional: you can save the OIDC configuration ID as a variable to use later. Run the following command to save the variable:
 +
 [source,terminal]
 ----
-$ export OIDC_ID=30f5dqmk
+$ export OIDC_ID=<oidc_config_id><1>
 ----
+<1> In the example output above, the OIDC configuration ID is 13cdr6b.
 
-. View the value of the variable by running with the following command:
+** View the value of the variable by running the following command:
 +
 [source,terminal]
 ----
 $ echo $OIDC_ID
 ----
 +
-.Sample output
+.Example output
 +
 [source,terminal]
```

Comment:
Can you change this source to `text`?

Reply:
I am going to leave it as `[source, terminal]` per the [OpenShift contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets).
<img width="975" alt="Screenshot 2024-03-21 at 3 35 17 PM" src="https://github.com/openshift/openshift-docs/assets/119696027/608a17a5-6887-4397-a392-07beba306a32">